### PR TITLE
903354 - UG Documentation for SfScheduler

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
Added ug documentation for SfScheduler

### CODE REVIEW

File: MAUI/Scheduler/appointments.md [hunk 1]
```diff
@@ -1051,6 +1101,85 @@ N>
 * The `BindingContext` for `AppointmentTemplate` for both `SchedulerAppointment` and `SchedulerAppointment.DataItem` in `AppointmentsSource.`
 * When using data template selector, performance issues occur as the conversion template views take time within the framework.
 
+## Commands
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/appointments.md:1104]: add the command topic in the events

File: MAUI/Scheduler/events.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ Below is a list of the arguments:
     * [Appointments](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerInteractionEventArgs.html#Syncfusion_Maui_Scheduler_SchedulerInteractionEventArgs_Appointments): Returns the selected appointments.
     * [Date](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerInteractionEventArgs.html#Syncfusion_Maui_Scheduler_SchedulerInteractionEventArgs_Date) : Returns the selected date.
     * [Element](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerInteractionEventArgs.html#Syncfusion_Maui_Scheduler_SchedulerInteractionEventArgs_Element) : Returns the `Scheduler` element tapped.
+    * `WeekNumber` : Returns the selected week number value.
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/events.md:26]: modify the content properly

File: MAUI/Scheduler/getting-started.md [hunk 3]
```diff
@@ -574,6 +574,25 @@ N> This property will not be applicable for the `SchedulerView` is `Timeline Mon
 
 ![show-week-number-in-maui-scheduler](images/getting-started/show-week-number-in-maui-scheduler.png)
 
+## Show Today button
+
+By Using the `ShowTodayButton` property of the [SfScheduler](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SfScheduler.html), you can show or hide the Today button. This support is to help users to show or hide the Today button dynamically irrespective to the Navigation arrows.
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/getting-started.md:579]: By Using the `ShowTodayButton` property of the [SfScheduler](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SfScheduler.html), you can show or hide the Today button. - after this explain default value and it functionality

File: MAUI/Scheduler/appointments.md (hunk 1 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/appointments.md]: Updated the changes

File: MAUI/Scheduler/getting-started.md (hunk 3 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/getting-started.md]: Updated teh changes

File: MAUI/Scheduler/appointments.md [hunk 4]
```diff
@@ -882,6 +882,56 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
 
+## Suspend and resume the appointment update
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/appointments.md:885]: Change this as - Suspend and resume the appointment UI rendering

File: MAUI/Scheduler/appointments.md [hunk 5]
```diff
@@ -882,6 +882,56 @@ N>
...
+
+Schedule allows you to suspend and resume the appointment UI update while performing collection changes (Add/Remove/Reset).`SuspendAppointmentViewUpdate ` method will suspend appointment UI rendering until you resume it when large number of data added dynamically in schedule `AppointmentSource` to avoid each time updating UI when collection changes. After data added dynamically in schedule, you can call `ResumeAppointmentViewUpdate ` to update the appointment UI rendering.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/appointments.md:887]: Schedule allows you to suspend and resume the appointment UI update while performing collection changes (Add/Remove/Reset) **to improve UI rendering when working with large appointment collections**

File: MAUI/Scheduler/appointments.md (hunk 4 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/appointments.md:885]: Updated the changes

File: MAUI/Scheduler/appointments.md (hunk 5 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/appointments.md]: updated the changes

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**